manifests/mod/php.pp

```diff
@@ -106,6 +106,14 @@
   }
 
   if $facts['os']['name'] == 'SLES' {
+    if $_package_name == 'apache2-mod_php7' and versioncmp($facts['os']['release']['major'], '0') == '15' {
```

Comment:
don't think this is the right usage of versioncmp, it returns an integer between -1 and 1.
you can either do:
```suggestion
    if $_package_name == 'apache2-mod_php7' and versioncmp($facts['os']['release']['major'], '15') >= 0 {
```
or like in ntp:
```suggestion
    if $_package_name == 'apache2-mod_php7' and $facts['os']['release']['major'] == '15' {
```

Reply:
`versioncmp($facts['os']['release']['major'], '15')` doesn't seems to be working, hence taking the ntp one :)